packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1501,18 +1507,19 @@ export default abstract class SqlIntegration
           "No database provided. Please edit the connection settings and try again."
         );
       }
-      path += database + ".";
+      path +=
+        this.tablePathEnclosure + database + this.tablePathEnclosure + ".";
     }
 
     // Add schema if required
-    if (this.requiresSchema) {
+    if (this.requiresSchema || requireSchema) {
```

Comment:
Not blocking, but perhaps there is a better variable name for `requireSchema`. Maybe `queryRequiresSchema` - it took me a minute to understand the different between `requireSchema` and `this.requiresSchema`.